web/src/user_card_popover.js

```diff
@@ -785,13 +785,17 @@ function register_click_handlers() {
         e.preventDefault();
     });
 
-    $("body").on("click", ".view_user_profile", (e) => {
-        const user_id = Number.parseInt($(e.currentTarget).attr("data-user-id"), 10);
-        const user = people.get_by_user_id(user_id);
-        toggle_user_card_popover(e.target, user);
-        e.stopPropagation();
-        e.preventDefault();
-    });
+    $("body").on(
+        "click",
+        ".view_user_profile, .pill-container.person_picker > .pill[data-user-id]",
```

Comment:
In the original PR, there's new separate function responsible for adding event listener to the pill on stream and group setting. In the original PR, it was on line 885 in `web/src/user_card_popover.js`. However, it introduced a new bug because the pill container targeted by the event listener this comment is on is also activated and ended up popping up double popover when clicked. So, I added a second selector to this function instead.

_double popover bug:_
![Screenshot from 2024-04-30 20-46-58](https://github.com/zulip/zulip/assets/108744694/4734f461-5c64-4ab3-a31a-f05267cc8152)